zha/zigbee/cluster_handlers/homeautomation.py

```diff
@@ -65,10 +65,26 @@ class MeasurementType(enum.IntFlag):
             attr=ElectricalMeasurement.AttributeDefs.active_power.name,
             config=REPORT_CONFIG_OP,
         ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.active_power_ph_b.name,
+            config=REPORT_CONFIG_OP,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.active_power_ph_c.name,
+            config=REPORT_CONFIG_OP,
+        ),
         AttrReportConfig(
             attr=ElectricalMeasurement.AttributeDefs.active_power_max.name,
             config=REPORT_CONFIG_DEFAULT,
         ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.active_power_max_ph_b.name,
+            config=REPORT_CONFIG_DEFAULT,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.active_power_max_ph_c.name,
+            config=REPORT_CONFIG_DEFAULT,
+        ),
```

Comment:
The `ActivePowerMax` attributes also don't support attribute reporting per spec.